src/main/java/org/openelisglobal/config/AppConfig.java

```diff
@@ -58,6 +56,10 @@
 @PropertySource("classpath:application.properties")
 @PropertySource("file:/run/secrets/common.properties")
 @PropertySource(value = "file:/run/secrets/extra.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "classpath:SystemConfiguration.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "file:/var/lib/openelis-global/properties/SystemConfiguration.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "file:/var/lib/openelis-global/properties/TotalSystemConfiguration.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "file:/var/lib/openelis-global/properties/SystemConfiguration.properties", ignoreResourceNotFound = true)
```

Comment:
i see this is a duplicate , did you mean to put  `ChangedSystemConfiguration.properties` here ??

Reply:
Nope, a true duplicate. Will fix